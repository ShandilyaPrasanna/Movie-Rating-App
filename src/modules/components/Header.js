import React from "react";
import { makeStyles } from "@material-ui/core/styles";
import AppBar from "@material-ui/core/AppBar";
import Toolbar from "@material-ui/core/Toolbar";
import Typography from "@material-ui/core/Typography";
import IconButton from "@material-ui/core/IconButton";
import logo from "../../assets/img/logo.png";

//Header Component
const useStyles = makeStyles(theme => ({
  logo: {
    marginLeft: "50%",
    transform: "translateX(-50%)"
  }
}));

export default function Header() {
  const classes = useStyles();
  return (
    <div style={{ margin: "30px" }}>
      <img src={logo} height="50px" width="200px" className={classes.logo} />
    </div>
  );
}
