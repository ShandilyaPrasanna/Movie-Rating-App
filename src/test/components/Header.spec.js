import React from "react";
import ReactDOM from "react-dom";
import toJson from "enzyme-to-json";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import Header from ".../../modules/components/Header";

jest.mock("react-dom");

configure({ adapter: new Adapter() });
describe("Header Test", () => {
  let tree;
  beforeEach(() => {
    tree = shallow(<Header />);
  });

  it("renders Header ", () => {
    expect(toJson(tree)).toMatchSnapshot();
  });
});
