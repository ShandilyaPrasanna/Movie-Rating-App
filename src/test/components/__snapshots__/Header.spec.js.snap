// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Test renders Header  1`] = `
<div
  style={
    Object {
      "margin": "30px",
    }
  }
>
  <img
    className="makeStyles-logo-1"
    height="50px"
    src="logo.png"
    width="200px"
  />
</div>
`;
